apps/root/src/pages/strategy-guardian-detail/frame/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Grid, BackControl, ContainerBox } from 'ui-library';
+import { useParams } from 'react-router-dom';
+import usePushToHistory from '@hooks/usePushToHistory';
+import { useAppDispatch } from '@state/hooks';
+import useTrackEvent from '@hooks/useTrackEvent';
+import { defineMessage, useIntl } from 'react-intl';
+import { changeRoute } from '@state/tabs/actions';
+import { DCA_ROUTE, EARN_ROUTE } from '@constants/routes';
+import NetWorth from '@common/components/net-worth';
+import EarnFAQ from '@pages/earn/components/faq';
+
+const StrategyDetailFrame = () => {
+  const { chainId } = useParams<{
+    strategyGuardianId: string;
+    chainId: string;
+  }>();
+  const pushToHistory = usePushToHistory();
+  const dispatch = useAppDispatch();
+  const trackEvent = useTrackEvent();
+  const intl = useIntl();
+
+  React.useEffect(() => {
+    dispatch(changeRoute(EARN_ROUTE.key));
```

Comment:
This one would actually change based on if the user had an investment there right? Since we would like to highlight the "portfolio" sub item

Reply:
So true!